tap_asana/streams/portfolios.py

```diff
@@ -1,21 +1,36 @@
 
 from singer import utils
 from tap_asana.context import Context
-from tap_asana.streams.base import Stream, asana_error_handling
+from tap_asana.streams.base import Stream, asana_error_handling, REQUEST_TIMEOUT
 
 
 @asana_error_handling
 def get_items_for_portfolio(portfolio_gid):
+  # Set request timeout to config param `request_timeout` value.
+  config_request_timeout = Context.config.get('request_timeout')
+  if config_request_timeout and float(config_request_timeout):
+      request_timeout = float(config_request_timeout)
+  else:
+      request_timeout = REQUEST_TIMEOUT # If value is 0,"0","" or not passed then it set default to 300 seconds.
```

Comment:
Put this comment above the if condition.

Reply:
Done